app/lib/session.js

```diff
@@ -3,10 +3,9 @@
 const session = require('express-session');
 const RedisStore = require('connect-redis')(session);
 const Redis = require('./redis');
+const linzDefaults = require('linz/lib/defaults');
 
-module.exports = session({
-    resave: false,
-    saveUninitialized: false,
-    secret: 'sessionsecret',
-    store: new RedisStore({ client: new Redis() }),
-});
+const store = new RedisStore({ client: new Redis() });
+const sessionOptions = { ...linzDefaults['session options'], ...{ store } };
```

Comment:
@smebberson I don't think we specify which node version we support anywhere, but won't this only work with node 8+?

Reply:
That code, yes. But it's just the example app. I've just added a `engine.node` entry to `package.json` accordingly.

Reply:
@allanchau, that okay?

Reply:
Yep sounds good.